roofit/roofitcore/src/RooAbsData.cxx

```diff
@@ -173,7 +173,7 @@ bool RooAbsData::releaseVars(RooAbsData* data)
 RooAbsData::RooAbsData()
 {
   claimVars(this) ;
-  _dstore = 0 ;
+  _dstore = nullptr;
```

Comment:
Consider using this in the header
```diff
- RooAbsDataStore* _dstore ; ///< Data storage implementation
+ RooAbsDataStore* _dstore = nullptr; ///< Data storage implementation
```
and removing all inits from the constructors.